ansible/eos_install_config

```diff
@@ -107,7 +124,8 @@ def main():
             password=dict(required=True),
             use_ssl=dict(required=False, default=True),
             config_file=dict(required=True),
-            commit_changes=dict(required=True),
+            commit_changes=dict(required=False, default=False),
+            replace_config=dict(required=False, default=False),
```

Comment:
But it won't do anything, if you will want for your change to take effect, you need to specify that you want that explicitly